chain/client/src/sync.rs

```diff
@@ -672,6 +672,21 @@ impl StateSync {
         Ok((update_sync_status, all_done))
     }
 
+    fn get_epoch_start_sync_hash(
+        &self,
+        chain: &mut Chain,
+        sync_hash: CryptoHash,
```

Comment:
why not pass as reference to avoid unnecessary copying?

Reply:
++

Reply:
@bowenwang1996 I found that I don't understand what are you talking about. What are we saving at this point?

Reply:
I've changed to `sync_hash: &CryptoHash` as you suggested, however, it doesn't make any sense to me.